pandas-stubs/core/frame.pyi

```diff
@@ -21,26 +21,29 @@ import numpy as _np
 
 from .groupby.generic import DataFrameGroupBy, SeriesGroupBy
 from .indexes import Index
-from .indexing import _iLocIndexerFrame, _LocIndexerFrame
+from .indexing import _iLocIndexerFrame, _LocIndexerFrame, _AtIndexerFrame
 from .series import Series, _DTypeNp
 from .strings import StringMethods
 
 _str = str  # needed because DataFrame has a property called "str"...
 
 _AxisType = Literal["columns", "index"]
 
+_ErrorType = Literal['raise', 'ignore']
+
 _ListLike = Union[Series, Index, _np.ndarray, Sequence]
 
 _DType = TypeVar("_DType", bound=_np.dtype)
 
-_ColSubsetType = Union[Series, DataFrame, List[_str], _str, _np.ndarray[_np.str_]]
+_ColSubsetType = Union[Series, 'DataFrame', List[_str], _str, _np.ndarray[_np.str_]]
```

Comment:
This quoting shouldn't be needed in type stub files. mypy has multiple parsing phases, so it doesn't matter whether `DataFrame` has not been defined at this point. Did you get an error here?

```suggestion
_ColSubsetType = Union[Series, DataFrame, List[_str], _str, _np.ndarray[_np.str_]]
```